pkg/workload/workload.go

```diff
@@ -43,6 +43,13 @@ import (
 	utilmaps "sigs.k8s.io/kueue/pkg/util/maps"
 )
 
+const (
+	PendingStatus       = "pending"
```

Comment:
I think 
```suggestion
	StatusPending       = "pending"
```
sounds a bit better. Same for the others.